visidata/main.py

```diff
@@ -81,6 +82,14 @@ def optalias(abbr, name, val=None):
 
 def main_vd():
     'Open the given sources using the VisiData interface.'
+    if any(i in ['-v', '--version'] for i in sys.argv):
```

Comment:
Let's invert it: `any(x in sys.argv for x in ['-v', '--version']`.   Or since there are only two, probably cleaner to use `'-v' in sys.argv or '--version' in sys.argv`.

Reply:
My heart for a "if (x or y) in z".